util/gitutil/git_commit.go

```diff
@@ -0,0 +1,9 @@
+package gitutil
+
+import "regexp"
+
+var validHex = regexp.MustCompile(`^[a-f0-9]{40}$`)
+
+func IsCommitSHA(str string) bool {
+	return validHex.MatchString(str)
```

Comment:
(this is refactoring of old code, but) This is simple enough that I think regexp can be avoided (especially because it runs in `init()` phase)

Reply:
Changed into a more manual check to avoid the `regexp`.